packages/wonder-blocks-switch/src/themes/default.ts

```diff
@@ -0,0 +1,66 @@
+import {tokens} from "@khanacademy/wonder-blocks-theming";
+
+const theme = {
+    color: {
+        bg: {
+            primary: {
+                off: tokens.color.offBlack50,
+                disabledOff: tokens.color.offBlack32,
+                activeOff: tokens.color.offBlack64,
+                on: tokens.color.blue,
+                disabledOn: tokens.color.fadedBlue,
+                activeOn: tokens.color.activeBlue,
+            },
+            secondary: {
+                on: tokens.color.white,
+                off: tokens.color.white,
+            },
+            tertiary: {
+                on: tokens.color.blue,
+                disabledOn: tokens.color.fadedBlue,
+                off: tokens.color.offBlack50,
+                disabledOff: tokens.color.offBlack32,
+            },
+        },
+        outline: {
+            default: tokens.color.blue,
+        },
+    },
+    border: {
+        radius: {
+            // slider
+            small: tokens.spacing.small_12,
+            // switch
+            full: tokens.border.radius.full,
+        },
+    },
+    size: {
+        height: {
+            none: 0,
+            // switch
+            medium: 20,
+            // slider
+            large: tokens.spacing.large_24,
+        },
+        width: {
+            none: 0,
+            xxxxSmall: tokens.spacing.xxxxSmall_2,
```

Comment:
**suggestion:** It would be better to try to dissociate the value from the token naming.

Instead of `xxxxSmall: tokens.spacing.xxxxSmall_2`, this could be something like `small: tokens.spacing.xxxxSmall_2`.

**Why:**
Using this approach would help to have a more flexible naming in case we want to provide a different value in a different theme.

For example, in the future, we want to provide a different `width.small` value to the `khanmigo` theme (e.g. `8px`), so instead of enforcing this to a `xxxxSmall` size, we can group this into a more flexible `small` size.

Reply:
that makes sense! i didn't think about how using a new theme would affect the naming.